src/components/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+
+import Home from './Home';
+import Contacts from './Contacts';
+import Header from './Header';
+import Menu from './Menu';
+import { NAV } from '../constants/constants';
+
```

Comment:
One extra blank line is enough between class component and imports. Do that for all the components.